src/components/ClassroomButton.astro

```diff
@@ -13,24 +13,34 @@ interface Props {
 }
 
 const { text, color, href } = Astro.props;
-
+let colorHover: string;
 let colorClass: string;
+/* hover colors to be changed*/
 switch (color) {
   case ClassroomButtonColor.paleYellow:
-    colorClass = "bg-pale-yellow";
+    colorClass = "bg-pale-yellow ";
+    colorHover = "hover:text-orange ";
```

Comment:
it is all the same, why not make it a const above or just put it directly in the html code?